src/core/cgroup.c

```diff
@@ -4167,6 +4167,7 @@ int compare_job_priority(const void *a, const void *b) {
 int unit_cgroup_freezer_action(Unit *u, FreezerAction action) {
         _cleanup_free_ char *path = NULL;
         FreezerState target, kernel = _FREEZER_STATE_INVALID;
+        Unit *slice;
```

Comment:
move this inline with the initialization

Reply:
I meant in the block where it's used - we declare variables in the smallest scope possible as per coding style guide

Reply:
Done.